ui/page/settings/view.jsx

```diff
@@ -259,7 +281,30 @@ class SettingsPage extends React.PureComponent<Props, State> {
                 </React.Fragment>
               }
             />
-
+            {/* @if TARGET='app' */}
```

Comment:
Maybe this should be under "extra settings". We could toggle showing that if the file ends in certain extensions, or over a certain size?